mava/components/training/step.py

```diff
@@ -240,6 +241,17 @@ def sgd_step(
                 data.extras,
             )
 
+            # Perform observation normalization if neccesary before proceeding
+            observation_stats = states.observation_stats
+            if hasattr(trainer.store, "norm_obs_running_stats_fn"):
```

Comment:
I think this might be a recurring thing, i.e. that we need to check if something is in the store or not. Perhaps, we can think about adding an easy method to the store class to check this, e.g. `store.has("something")`, and for this `hasattr` should work, but then at least it looks much cleaner in other parts of the code and is shorter than checking the config.

Reply:
I just adopted @sash-a approach. Here we actually really don't need since we can just use` if trainer.store.global_config.normalize_observations or if trainer.store.global_config.normalize_target_values`